app/components/alert_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= tag.div(**tag_options, class: css_class, role:) do %>
+<%= tag.div(role:, **tag_options, class: css_class) do %>
```

Comment:
It'd be good to include some test coverage for this change, specifically that it's possible to override the default `role` with another value.